src/examples/sparqlTurtleGraph.ts

```diff
@@ -0,0 +1,98 @@
+import { createElement, ClassAttributes } from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { Workspace, WorkspaceProps } from '../index';
+
+import { onPageLoad } from './common';
+import { RDFDataProvider } from '../ontodia/data/rdf/rdfDataProvider';
+import GraphBuilder from "../ontodia/data/sparql/graphBuilder";
```

Comment:
Please don't use default imports if possible (it gets pretty hard to refactor imports like this).